### PR TITLE
[BPK-1494] "updated at" format fix

### PR DESCRIPTION
**Pull request by matteo-hertel**
tested on the regular docs, SSR, and SSR with disabled JS

### CODE REVIEW

**Comment by matthewdavidson** [comment]
@matteo-hertel we have `date-fns` as a dependency :) use that instead.

**Comment by matteo-hertel** [comment]
damn

**Comment by matteo-hertel** [comment]
damn

File: packages/bpk-docs/src/pages/NeoHomePage/UpdatedAt.js [hunk 1]
```diff
@@ -19,16 +19,12 @@
...
+import { format } from 'date-fns';
 
-const formatDate = ((): (Date => string) => {
-  if (typeof window === 'object' && window.Intl && window.Intl.DateTimeFormat) {
-    const formatter = new window.Intl.DateTimeFormat();
-    return (date: Date): string => formatter.format(date);
-  }
+import BpkText from 'bpk-component-text';
 
-  return (date: Date): string => date.toLocaleDateString();
```
Comment (k0nserv) [inline packages/bpk-docs/src/pages/NeoHomePage/UpdatedAt.js]: You can just force a locale with `toLocaleDateString('en-GB')` can't you?

**Comment by matteo-hertel** [comment]
ah! I'll add that in, lesson for me, don't blindly trust comments on github 😄

**Comment by matteo-hertel** [comment]
third commit the charm 🤞, should be good to go now

File: packages/bpk-docs/src/pages/NeoHomePage/UpdatedAt.js [hunk 2]
```diff
@@ -19,16 +19,12 @@
...
-})();
+const formatDate = ((): (Date => string) => (date: Date): string =>
```
Comment (k0nserv) [inline packages/bpk-docs/src/pages/NeoHomePage/UpdatedAt.js]: We don't need the self executing function anymore. Just 

```javascripnt
const formatDate = (date: Date): string => format(date, 'DD/MM/YYYY')
```

Is enough

**Comment by matthewdavidson** [comment]
> We don't need the self executing function anymore. Just
>`const formatDate = (date: Date): string => format(date, 'DD/MM/YYYY')`
>Is enough

@matteo-hertel / @k0nserv  i pushed a fix for this. Please approve

**Comment by matteo-hertel** [comment]
I can't approve as it's my PR but  LGTM

**Pull request merged**